action/main.py

```diff
@@ -20,7 +20,7 @@
         sys.exit(1)
     version_specifier = f"=={VERSION}"
 
-req = f"ruff{version_specifier}"
+req = f"ruff{version_specifier} check"
```

Comment:
Instead of adding `"check"` here, it feels more correct to include it below in the argument list:

```python
proc = run(["pipx", "run", req, "check", *shlex.split(ARGS), *files_to_check])
```

What do you think?